components/brave_wallet/browser/brave_wallet_provider_impl.cc

```diff
@@ -7,15 +7,26 @@
 
 #include <utility>
 
+#include "base/json/json_writer.h"
+#include "base/strings/stringprintf.h"
 #include "brave/components/brave_wallet/browser/brave_wallet_provider_delegate.h"
+#include "brave/components/brave_wallet/browser/brave_wallet_utils.h"
 #include "brave/components/brave_wallet/browser/eth_json_rpc_controller.h"
+#include "brave/components/brave_wallet/browser/eth_response_parser.h"
+#include "components/user_prefs/user_prefs.h"
+
+namespace {
+const char kAddEthereumChainMethod[] = "wallet_addEthereumChain";
+const char kJsonResponseF[] = R"({"id":1,"jsonrpc":"2.0",%s})";
```

Comment:
I would add all constants into the `brave/components/brave_wallet/common/web3_provider_constants.h`

Reply:
done